### PR TITLE
make sure the element exists before styling it

### PR DESCRIPTION
**Pull request by thijstriemstra**
Sometimes right after destroying a wavesurfer, an additional `drawer.style` call is made somewhere but since the element does not exist, it throws an error in `drawer.js`:

```
TypeError: el is null

if (el.style[prop] != styles[prop]) {
```

This PR makes sure the element exists before styling it.

### CODE REVIEW

**Comment by thijstriemstra** [comment]
@katspaugh could you also release a new version after merging this, it's been a while.. :)

File: src/drawer.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ WaveSurfer.Drawer = {
 
     style: function (el, styles) {
```
Comment (katspaugh) [inline src/drawer.js]: It would be better to check if element exists here, and exit early if not.

**Comment by katspaugh** [comment]
Have you figured out why is that extra call to style is made? The must be async callback somewhere.

**Comment by thijstriemstra** [comment]
Ok I'm using the microphone plugin in a Bootstrap modal. When you open the modal the microphone plugin is displayed, press the play button, and your microphone will start emitting a signal that is drawn by wavesurfer. When you press the close button on the modal I destroy the wavesurfer instance.

What the microphone plugin does, is basically continously calling `reloadBuffer`:

```javascript
/**
 * Redraw the waveform.
 */
reloadBuffer: function(event) {
    if (!this.paused) {
        this.wavesurfer.empty();
        this.wavesurfer.loadDecodedBuffer(event.inputBuffer);
    }
},
```
So I expect that that wavesurfer `empty` and `loadDecodedBuffer` are still being called after `destroy` is called on it. So I guess making sure to set `this.paused` to `true` just before destroying the microphone plugin (and it's associated wavesurfer instance) should be enough to fix it. Let me check.

**Comment by katspaugh** [comment]
So 881b8fc (checking if element exists) is not really needed? I'd prefer not to check and let it throw then, because otherwise we won't know of possible bugs.

**Comment by thijstriemstra** [comment]
@katspaugh, yeah that's right, fine with me.

**Comment by katspaugh** [comment]
OK, I rolled that commit back. Thanks!

**Pull request merged**